doc/classes/PhysicsBody2D.xml

```diff
@@ -48,11 +51,14 @@
 			<param index="1" name="distance" type="Vector2" />
 			<param index="2" name="collision" type="KinematicCollision2D" default="null" />
 			<param index="3" name="safe_margin" type="float" default="0.08" />
+			<param index="4" name="recovery_as_collision" type="bool" default="false" />
 			<description>
 				Checks for collisions without moving the body. In order to be frame rate independent in [method Node._physics_process] or [method Node._process], [param distance] should be computed using [code]delta[/code].
 				Virtually sets the node's position, scale and rotation to that of the given [Transform2D], then tries to move the body along the vector [param distance]. Returns [code]true[/code] if a collision would stop the body from moving along the whole path.
```

Comment:
I propose to add something directly after this line, because with the new parameter it is no longer the whole truth. Something like `If [param recovery_as_collision] is [code]true[/code], any depenetration from the recovery phase is also counted as a collision; this is useful for checking whether the body would [i]touch[/i] any other bodies.`